service_mesh/service_mesh_install/updating-ossm.adoc

```diff
@@ -0,0 +1,44 @@
+[id="updating-ossm"]
+= Updating {ProductName} from version 1.0.1 to 1.0.2
+include::modules/ossm-document-attributes.adoc[]
+:context: installing-ossm
+toc::[]
+
+Updating {ProductName} requires extra steps before you update {product-title} to version 4.2. 
+You must upgrade {ProductName} to 1.0.2 before upgrading {product-title} 4.1.x to 4.2.
+
+.Prerequisites
+
+* {ProductName} version 1.0.1
+* {product-title} version 4.1
+
+.Procedure
+
+. Configure existing SMCP resource requests by running the following `oc patch` command. Replace
+the <smcp_namespace> and <smcp_name> with your specific names:
++
+----
+$ oc patch -n <smcp_namespace> smcp <smcp_name> \// <1>
+    --type=merge -p \
+    '{"spec": {"istio": {"global": {"defaultResources": {"requests": {"cpu": "10m","memory": "128Mi"},"limits":{}},"proxy": {"resources": {"requests": {"cpu": "10m","memory": "128Mi"},"limits":{}}},"defaultPodDisruptionBudget": {"enabled": false}},"security": {"resources": {"requests": {"cpu": "10m","memory": "128Mi"}}},"galley": {"resources": {"requests": {"cpu": "10m","memory": "128Mi"}}},"pilot": {"resources": {"requests": {"cpu": "10m","memory": "128Mi"}}},"mixer": {"telemetry": {"resources": {"requests": {"cpu": "10m","memory": "128Mi"}}}},"gateways": {"istio-egressgateway": {"resources": {"requests": {"cpu": "10m","memory": "128Mi"}}},"istio-ingressgateway": {"resources": {"requests": {"cpu": "10m","memory": "128Mi"}}}},"prometheus": {"resources": {"requests": {"cpu": "10m","memory": "128Mi"}}}}}}'
+----
+<1> For example, `basic-install`.
++
+After running this command, wait until all SMCP Pods are replaced in the SMCP namespace.
++
+. After the Pods are running in the SMCP namespace, redeploy your Data Plane applications, such as `bookinfo`.  
++
+. Log in as a `cluster-admin` user such as `kubeadmin`, and then run the following command to delete the CNI `istio-node` DaemonSet.  
+Replace `openshift-operators` if your {ProductName} Operator was not installed in the default `openshift-operators` namespace:
++
+----
+$ oc delete -n openshift-operators daemonset istio-node
+----
++
+. Upgrade {ProductName} Operator and SMCP to 1.0.2. After all Pods are running in the SMCP namespace, patch Data Plane applications by running the following command for each deployment:
++
+----
+$ oc patch -n <Data Plane namespace> deployment/<deployment_name> -p '{"spec":{"template":{"metadata":{"annotations":{"kubectl.kubernetes.io/restartedAt": "'`date -Iseconds`'"}}}}}'
```

Comment:
Oops, sorry, I thought I entered a comment here. Same deal with the replaceables (lowercase w/ underscores), and consider line-breaking:

```
----
$ oc patch -n <data_plane_namespace> deployment/<deployment_name> -p \
    '{"spec":{"template":{"metadata":{"annotations":{"kubectl.kubernetes.io/restartedAt": "'`date -Iseconds`'"}}}}}'
----
```